### ISSUE TITLE
Helpers cannot be resolved

### ISSUE THREAD

**Issue by eroux** [BuddhistDigitalResourceCenter/lds-pdi#31]
Looking at the logs, I see some:

```
An error occurred at line: 94 in the jsp file: /ontRes.jsp
Helpers cannot be resolved
91:                 <h4>Subclasses:</h4>
92:             
93:                 <%for(OntClassModel sub:subclasses){%>
94:                 <ul><li><a href="<%=Helpers.relativizeURL(sub.getUri())%>"><%=sub.getId() %></a></li></ul>
95:             <%}}%>              
96: <% } else {%>
97: <p>This class is defined external to this ontology.</p>
```

probably linked to the renaming of `Helpers` into `helpers`. A good set of unit tests would help prevent this kind of errors

**Comment by MarcAgate** [comment]
Fixed as of commit 11ab622

**Comment by eroux** [comment]
I can't see anything in the logs anymore (although I can't really remember which log it was), closing
